spec/forms/schools/setup_school_cohort_form_spec.rb

```diff
@@ -4,6 +4,19 @@
   describe "validations" do
     it { is_expected.to validate_presence_of(:expect_any_ects_choice).on(:expect_any_ects) }
     it { is_expected.to validate_presence_of(:how_will_you_run_training_choice).on(:how_will_you_run_training) }
+    it { is_expected.to validate_presence_of(:change_provider_choice).on(:change_provider) }
+    it { is_expected.to validate_presence_of(:what_changes_choice).on(:what_changes) }
+  end
+
+  describe ".PROGRAMME_CHOICES_MAP" do
+    it "returns a hash with the user programme choices and their respective programme types" do
+      expect(described_class::PROGRAMME_CHOICES_MAP).to eq({
+        "change_lead_provider" => "full_induction_programme",
+        "change_delivery_partner" => "full_induction_programme",
+        "change_to_core_induction_programme" => "core_induction_programme",
+        "change_to_design_our_own" => "design_our_own",
+      })
+    end
```

Comment:
I'm not sure it's worth replicating all the data from the classes to test it.

Reply:
My thought is that we would need to protect the integrity of the map. Without it, you could change for example the `change_lead_provider` to point `core_induction_programme` and get unexpected results without noticing.